src/wallet.js

```diff
@@ -549,56 +569,81 @@ export class Wallet extends React.Component {
 
     return (
       <div>
-        <div style={{width: '100%', textAlign: 'right'}}>
-          <Button onClick={() => this.setState({settingsModal: true})}>
-            <Glyphicon
-              style={{backgroundColor: 'white'}}
-              glyph="menu-hamburger"
-            />
-          </Button>
-        </div>
         {busyModal}
         {settingsModal}
-        <DismissibleMessages
-          messages={this.state.messages}
-          onDismiss={index => this.dismissMessage(index)}
-        />
-        <Well>
-          <FormGroup>
-            <ControlLabel>Account Public Key</ControlLabel>
-            <InputGroup>
-              <FormControl
-                readOnly
-                type="text"
-                size="21"
-                value={this.state.account.publicKey}
-              />
-              <InputGroup.Button>
-                <OverlayTrigger placement="bottom" overlay={copyTooltip}>
-                  <Button onClick={() => this.copyPublicKey()}>
-                    <Glyphicon glyph="copy" />
-                  </Button>
-                </OverlayTrigger>
-              </InputGroup.Button>
-            </InputGroup>
-          </FormGroup>
-          <p />
-          Account Balance: {this.state.balance} &nbsp;
-          <OverlayTrigger placement="top" overlay={refreshBalanceTooltip}>
-            <Button onClick={() => this.refreshBalance()}>
-              <Glyphicon glyph="refresh" />
-            </Button>
-          </OverlayTrigger>
-          <OverlayTrigger placement="bottom" overlay={airdropTooltip}>
-            <Button
-              className="margin-left"
-              disabled={airdropDisabled}
-              onClick={() => this.requestAirdrop()}
-            >
-              <Glyphicon glyph="send" />
-            </Button>
-          </OverlayTrigger>
-        </Well>
+        <div className="container">
+          <DismissibleMessages
+            messages={this.state.messages}
+            onDismiss={index => this.dismissMessage(index)}
+          />
+        </div>
+        <Grid>
+          <Row>
+            <Col xs={12}>
+              <div className="account-header">
+                <h2 className="decor">
+                  account information
+                </h2>
+                <button onClick={() => this.setState({settingsModal: true})}>
+                  <span><GearIcon /> <span>Settings</span></span>
+                </button>
+              </div>
+            </Col>
+          </Row>
+          <Row>
+            <Col xs={12} md={5}>
+              <Well>
+                <h4>
+                  Account Balance
+                </h4>
+                <div className="balance">
+                  <div className="balance-val">
+                    {this.state.balance}
+                  </div>
+                  <div className="balance-ttl">
+                    Sols
```

Comment:
Please update to Lamports

Reply:
I think we want to move toward using SOLs consistently across our user-facing products. But presumably might need a little backend updating.